frontend/src/app/patients/UploadPatients.tsx

```diff
@@ -281,14 +336,19 @@ const UploadPatients = ({ isAdmin }: { isAdmin: boolean }) => {
                             <FontAwesomeIcon icon={faXmark} />
                           </button>
                           <p className="usa-alert__text">
-                            {errorMessageText} See the{" "}
-                            <a
-                              target="_blank"
-                              href="/using-simplereport/manage-people-you-test/bulk-upload-patients/#preparing-your-spreadsheet-data"
-                            >
-                              patient bulk upload guide
-                            </a>{" "}
-                            for details about accepted values.
+                            {errorMessageText}
+                            {includeGuideInErrorMessage && (
+                              <div>
```

Comment:
div element needs to be switched to a span because is within a paragraph element. To make it behave like a div just add the class display-block
`<span class="display-block">`

Reply:
Thanks for catching this! Sharp eye 👀